bin/tictactoe

```diff
@@ -1,3 +1,137 @@
 #!/usr/bin/env ruby
-
 require_relative '../config/environment'
+
+
+
+## Methods for CLI ##
+
+    def player_options
+
+      puts "Welcome to OsTrItCh-TAC-TOE!"
+ 
+      puts <<-OSTRICH
+                                      (oo)
+                                          ||
+                                          ||
+                                          ||
+                                          ||,-v-,_
+                                          ||) |   /
+                                      _,''   '-,-<
+                                     / :       /  ) 
+                                    ( :       (   /,
+                                     )_;       ) __)
+                                        ),_ ,   |
+                                        |  / )  |
+                                        | /   ) |
+                                        ()     ()
+                                        //     ||
+                                       //      ||
+                                      //       ||
+                        -- '' -'-'  ,//        /)   -- '' -'- 
+
+      OSTRICH
```

Comment:
But can your ostrich do this?
![Ostriches are good skiers](http://i.giphy.com/zd8YN6oC3HCX6.gif)